homeassistant/components/frontend/__init__.py

```diff
@@ -135,7 +135,7 @@ def json(self) -> str:
         return self._serialized
 
     def _serialize(self) -> None:
-        self._serialized = json.dumps(self.manifest, sort_keys=True)
+        self._serialized = json_dumps_sorted(self.manifest)
```

Comment:
Would not make more sense to add sort_keys=True to our json_dump?

Reply:
I wanted to hide the implementation details of orjson from the rest of the code in case something changes in the future we would only have to change it in one spot.

Reply:
I mean, we have our own json_dumps function, we can spend them a sort_keys equivalent json.dumps have, so it's more compatible with the python json instead to create a new function which is the same as json_dumps just with a flag more

Reply:
oh, I see you created already a bunch of json_ function